scality/metal-k8s
# Update third party

This PR updates the ES and Kubespray trees we vendor, and updates all Helm chart versions to the latest version of the charts.

Furthermore, it fixes a minor bug in the `check-vendor.py` script, and makes sure we disable strict DNS checks for Docker on `etcd` hosts, which was an oversight before (required when `etcd` hosts are not part of `k8s-cluster`).

Test plan:
- Deploy on 'new' VMs
- Check all pods are running
- Manual check of Grafana, K8s dashboard, Kibana
- Re-run playbook, ensure everything still running
